runtime/doc/gui_w32.txt

```diff
@@ -383,7 +383,7 @@ Note that a menu that starts with ']' will not be displayed.
 7. Command line arguments				*gui-w32-cmdargs*
 
 Analysis of a command line into parameters is not standardised in MS Windows.
-Gvim has to provide logic to analyse a command line.  This logic is likely to
+gvim has to provide logic to analyse a command line.  This logic is likely to
```

Comment:
I'm not English native, so I'm afraid that I'm saying something wrong, but starting a sentence with a lower case might be unnatural.